components/NavigationDrawer/src/MDCBottomDrawerViewController.m

```diff
@@ -249,6 +256,64 @@ - (void)bottomDrawerTopTransitionRatio:
   }
 }
 
+- (void)bottomDrawerPresentTransitionDidEnd:
+    (MDCBottomDrawerPresentationController *)presentationController {
+  _currentTransitionCoordinator = nil;
+  if ([self.delegate respondsToSelector:@selector(bottomDrawerControllerDidEndOpenTransition:)]) {
+    [self.delegate bottomDrawerControllerDidEndOpenTransition:self];
+  }
+}
+
+- (void)bottomDrawerDismissTransitionDidEnd:
+    (MDCBottomDrawerPresentationController *)presentationController {
+  _isDrawerClosed = YES;
+  _currentTransitionCoordinator = nil;
+  if ([self.delegate respondsToSelector:@selector(bottomDrawerControllerDidEndCloseTransition:)]) {
+    [self.delegate bottomDrawerControllerDidEndCloseTransition:self];
+  }
+}
+
+- (void)bottomDrawerPresentTransitionWillBegin:
+            (MDCBottomDrawerPresentationController *)presentationController
+                               withCoordinator:
+                                   (id<UIViewControllerTransitionCoordinator>)transitionCoordinator
+                                 targetYOffset:(CGFloat)targetYOffset {
+  _isDrawerClosed = NO;
+  _currentTransitionCoordinator = transitionCoordinator;
```

Comment:
looks like _currentTransitionCoordinator is set but never looked at, is it needed?

Reply:
remnant - will remove! thanks for catching!